flake.nix

```diff
@@ -38,13 +38,25 @@
       inputs.flake-utils.follows = "flake-utils";
     };
     lowrisc_sail = {
-      url = "github:lowrisc/sail?ref=lowrisc";
+      url = "github:lowrisc/sail?ref=ast_translate";
       flake = false;
     };
     lowrisc_sail_riscv = {
       url = "github:lowrisc/sail-riscv?ref=81a266b6f65365b34180af7b91708265da653878";
       flake = false;
     };
+    lowrisc_yosys_slang = {
+      url = "https://github.com/mndstrmr/yosys-slang";
```

Comment:
It would be good to move this under the lowrisc org at some point.

Reply:
We should open an issue to track this once this PR is merged.

Reply:
https://github.com/lowRISC/ibex/issues/2321